src/content/en/updates/2019/02/trusted-types.md

```diff
@@ -0,0 +1,224 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Trusted Types is a new experimental API available in Chrome that helps prevent DOM-Based Cross-Site Scripting in your applications.
+
+{# wf_updated_on: 2019-02-08 #}
+{# wf_published_on: 2019-02-06 #}
+{# wf_tags: news,security,trusted-types,origintrials #}
+{# wf_featured_image: /web/updates/images/generic/security.png #}
+{# wf_featured_snippet: Trusted Types is a new experimental API available in Chrome that helps prevent DOM-Based Cross-Site Scripting in your applications. #}
+{# wf_blink_components: N/A #}
+
+# Trusted Types help prevent Cross-Site Scripting {: .page-title }
+
+#### TL;DR {: #tldr .hide-from-toc }
+We've created a new experimental API that aims to prevent DOM-Based Cross
+Site Scripting in modern web applications.
+
+{% include "web/_shared/contributors/koto.html" %}
+
+<aside class="caution"> We’re currently working on the specification and
+implementation details for this API. We’ll keep this post updated as Trusted
+Types mature. </aside>
```

Comment:
I'd suggest adding an explicit last updated date in here, especially given this will be updated frequently.